docs/test-fairy/sdk/tf-production.md

```diff
@@ -8,11 +8,162 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-/FAQ/Running_TestFairy_In_Production.html
 
-/FAQ/How_to_disable_TestFairy_in_production.html
 
+Running TestFairy in production offers numerous benefits, such as gaining valuable insights into user behavior, detecting and resolving issues promptly, and continuously improving your app's performance. With TestFairy, you can proactively monitor your production environment, gather valuable data, and make informed decisions to deliver a superior app to your users.
 
-https://github.com/testfairy/docs/blob/master/docs/800_FAQ/700_Running_TestFairy_In_Production.md
+## Running TestFairy in Production
 
-https://github.com/testfairy/docs/blob/master/docs/800_FAQ/300_How_to_disable_TestFairy_in_production.md
\ No newline at end of file
+When running TestFairy in production, you may record sensitive data such as medical information, financial data or photos.
+
+Therefore it is important to follow these guidelines:
+
+1. On iOS, you **must** request explicit user consent before recording start.
+Please read carefully the [Apple guidelines](https://developer.apple.com/app-store/review/guidelines/) and pay special attention to section 2.5.14
+
+1. On Android, please call disableAutoUpdate() in order to comply with [Play Store Developer Distribution Agreement](https://play.google.com/about/developer-distribution-agreement.html).
+
+2. When recording sensitive data you **must** use TestFairy's [end-to-end encryption](/Security/End_to_End_Data_Encryption.html) with your own private keys, so that only your team will be able to see your sessions.
```

Comment:
```suggestion
2. When recording sensitive data, you **must** use TestFairy's [end-to-end encryption](https://docs.testfairy.com/Security/End_to_End_Data_Encryption.html) with your private keys so that only your team can see your sessions.
```